flocker/dockerplugin/_api.py

```diff
@@ -313,14 +314,19 @@ def got_state(datasets):
 
     @app.route("/VolumeDriver.Mount", methods=["POST"])
     @_endpoint(u"Mount")
-    def volumedriver_mount(self, Name):
+    def volumedriver_mount(self, Name, ID):
         """
         Move a volume with the given name to the current node and mount it.
 
         Since we need to return the filesystem path we wait until the
         dataset is mounted locally.
 
         :param unicode Name: The name of the volume.
+        :param string ID: A unique ID for caller that requested the mount
+
+        Right now we do nothing with ID, but it can be used to track
+        the mount calls when multiple containers are trying to mount
+        the same volume.
 
         :return: Result that includes the mountpoint.
```

Comment:
:question: Maybe double check whether the return format has changed. Do we have to include the supplied ID in the json that we return from the API. 
If so, then we'll have to also update the JSON schema files.